Description: do not install configuration files with cmake
Author: Daniel Kertesz <daniel@spatof.org>
--- a/data/CMakeLists.txt
+++ /dev/null
@@ -1,6 +0,0 @@
-# Only install example.chk and example.conf from this directory
-# NOTE: I would've had this just find all files in the directory, but that would include files not needed (like this file)
-set(DATA example.chk botserv.example.conf example.conf hostserv.example.conf modules.example.conf operserv.example.conf chanserv.example.conf global.example.conf memoserv.example.conf nickserv.example.conf chanstats.example.conf irc2sql.example.conf stats.standalone.example.conf)
-install(FILES ${DATA}
-  DESTINATION ${CONF_DIR}
-)
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -462,7 +462,6 @@
 endif(${Anope_SOURCE_DIR} STREQUAL ${Anope_BINARY_DIR})
 
 # Go into the following directories and run their CMakeLists.txt as well
-add_subdirectory(data)
 add_subdirectory(docs)
 add_subdirectory(language)
 add_subdirectory(src)
